hr_timesheet_sheet_attendance/__manifest__.py

```diff
@@ -1,10 +1,10 @@
 {
     "name": "HR Timesheet Sheet Attendance",
-    "version": "14.0.1.0.0",
+    "version": "15.0.1.0.0",
     "category": "Human Resources",
     "sequence": 80,
     "license": "AGPL-3",
-    "author": "BizzAppDev, Odoo Community Association (OCA)",
+    "author": "BizzAppDev, Nitrokey GmbH,Odoo Community Association (OCA)",
```

Comment:
These changes doesn't entitle you as author.

Reply:
> CI's are red. Some comments on the code:

Yes because hr_timesheet_sheet is not merged into v15.0 main branch so is there anything here I can take care of?